mariadb-11.5.yaml

```diff
@@ -187,6 +187,7 @@ subpackages:
     dependencies:
       provides:
         - mariadb-client=${{package.full-version}}
+        - mysql-client=${{package.full-version}}
```

Comment:
Hey @Dentrax, any more context on this one? Did we remove a duplicate mysql-client package and add this provides instead?

Reply:
Could we add a code comment as well to explain why this provides two separate names?